src/api/layers/GeoAdminLayer.class.js

```diff
@@ -53,6 +53,9 @@ export default class GeoAdminLayer extends AbstractLayer {
      *   metadata) are still loading. Default is `false`
      * @param {LayerTimeConfig | null} [layerData.timeConfig=null] Time series config (if
      *   available). Default is `null`
+     * @param {Boolean} [layerData.hasDescription=true] Define if this layer has a description that
```

Comment:
@pakb  does there already exist a ticket for this or should this remain hard coded?